File: swiftDemo/swiftDemo/day001/Demo004.swift
//
//  Demo004.swift
//  swiftDemo
//
//  Created by zhuxingyue on 2020/4/23.
//  Copyright © 2020 zhuxingyue. All rights reserved.
//

import Foundation

/*
 break
 continue
 */
class Demo004{
    func test(){
        for i in 1...10{
            if i == 5{
                continue
            }else if i == 8{
                break
            }else{
                print("\(i)\t",terminator:"")
            }
        }
    }
}

File: ccDemo/ccDemo/day001/demo004.c
//
//  demo004.c
//  ccDemo
//
//  Created by zhuxingyue on 2020/4/23.
//  Copyright © 2020 zhuxingyue. All rights reserved.
//

#include "day001.h"

/*
 break:终止当前循环;并退出
 continue:终止当前循环,进行下次循环
 */
void test4(){
    //1~10,输出所有数字除了5;遇到8终止;
    
    for(int i=1;i<=10;i++){
        //排除5
        if(i==5){
            continue;
        }else if(i==8){//8终止
            break;
        }else{
            printf("%d\t",i);
        }
    }
    
    printf("\n");
}

